pkgs/by-name/sq/squawk-chat/package.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, cmake
+, kconf
+, lmdb
+, qxmpp
+, boost
+, libomemo-c
+, pkg-config
+, libsForQt5
+, imagemagick
+
+, kwalletSupport ? false # secure password storage
+, kioSupport ? false # better show in folder action
+, kconfSupport ? false # system themes support
+}:
+
+stdenv.mkDerivation rec {
+  pname = "squawk-chat";
+  version = "0.2.3";
+
+  src = (fetchFromGitea {
+    domain = "git.macaw.me";
+    owner = "blue";
+    repo = "squawk";
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-dPeB2DOJ22shgeV6UBXb6Kcswre4q3CRaWogLsLyl2U=";
+  }).overrideAttrs (_: {
+    GIT_CONFIG_COUNT = 1;
+    GIT_CONFIG_KEY_0 = "url.https://git.macaw.me/.insteadOf";
+    GIT_CONFIG_VALUE_0 = "gitea@git.macaw.me:";
+  });
+
+  nativeBuildInputs = [ cmake boost libomemo-c pkg-config libsForQt5.wrapQtAppsHook imagemagick libsForQt5.kwallet libsForQt5.kio ];
+  buildInputs = [ libsForQt5.qt5.qtbase lmdb qxmpp ]
+    ++ lib.optionals kwalletSupport [ libsForQt5.kwallet ]
+    ++ lib.optionals kioSupport [ libsForQt5.kio ]
+    ++ lib.optionals kconfSupport [ kconf ];
+
+  cmakeFlags = [
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+    "-DCMAKE_INSTALL_INCLUDEDIR=include"
+    "-DCMAKE_INSTALL_BINDIR=bin"
+    "-DSYSTEM_LMDBAL=False"
+  ];
+
+  meta = with lib; {
+    description = "squawk";
+    homepage = "https://git.macaw.me/blue/squawk";
+    license = licenses.gpl3;
+    platforms = with platforms; linux;
+    maintainers = with maintainers; [ annoyingrains ];
+    mainProgram = "squawk";
```

```suggestion
    mainProgram = "squawk-chat";
```

@AnnoyingRain5 checks are failing, please look at `pkgs/by-name` check [details](https://github.com/NixOS/nixpkgs/actions/runs/7945784692/job/21692824839?pr=286873).

Looks like this package name is taken, i've found it via [search.nixos.org](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=squawk). I think it should be renamed, as well as executable should be renamed to avoid conflicts with existing package.